File: Contains_Duplicate/Contains_Duplicate.cpp
#include <iostream>
#include <vector>
#include <unordered_set>
#include <algorithm>

using namespace std;

class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        for(int i = 0; i < nums.size() - 1; i++)
            if(nums[i] == nums[i+1]) return true;
        return false;
    }
};

int main() {
    vector<int> nums = {1, 2, 3, 1}; // Example input
    Solution sol;
    bool result = sol.containsDuplicate(nums);
    cout << (result ? "true" : "false") << endl;  // Expected output: true
    return 0;
}

File: Contains_Duplicate/Contains_Duplicate.py
from typing import List


def containsDuplicate(nums: List[int]) -> bool:
    nums.sort()
    for i in range(len(nums) - 1):
        if nums[i] == nums[i + 1]:
            return True
    return False


nums = [1, 2, 3, 1]  
result = containsDuplicate(nums)
print(result)  